### PR TITLE
Upgrade kube depedencies to 1.15.10

### PR DESCRIPTION
**Pull request by sayboras**
### Description

Fixes https://github.com/weaveworks/eksctl/issues/1965
Fixes https://github.com/weaveworks/eksctl/issues/1987

<details>
<summary>Create new cluster with default version (1.14) </summary>

```
$ ./eksctl create cluster 
[ℹ]  eksctl version 0.17.0-dev+fe5ebf04.2020-03-31T20:43:57Z
[ℹ]  using region us-east-2
[ℹ]  setting availability zones to [us-east-2b us-east-2c us-east-2a]
[ℹ]  subnets for us-east-2b - public:192.168.0.0/19 private:192.168.96.0/19
[ℹ]  subnets for us-east-2c - public:192.168.32.0/19 private:192.168.128.0/19
[ℹ]  subnets for us-east-2a - public:192.168.64.0/19 private:192.168.160.0/19
[ℹ]  nodegroup "ng-22e7fed7" will use "ami-020a35f771eac7c58" [AmazonLinux2/1.14]
[ℹ]  using Kubernetes version 1.14
[ℹ]  creating EKS cluster "exciting-monster-1585647870" in "us-east-2" region with un-managed nodes
[ℹ]  will create 2 separate CloudFormation stacks for cluster itself and the initial nodegroup
[ℹ]  if you encounter any issues, check CloudFormation console or try 'eksctl utils describe-stacks --region=us-east-2 --cluster=exciting-monster-1585647870'
[ℹ]  CloudWatch logging will not be enabled for cluster "exciting-monster-1585647870" in "us-east-2"
[ℹ]  you can enable it with 'eksctl utils update-cluster-logging --region=us-east-2 --cluster=exciting-monster-1585647870'
[ℹ]  Kubernetes API endpoint access will use default of {publicAccess=true, privateAccess=false} for cluster "exciting-monster-1585647870" in "us-east-2"
[ℹ]  2 sequential tasks: { create cluster control plane "exciting-monster-1585647870", create nodegroup "ng-22e7fed7" }
[ℹ]  building cluster stack "eksctl-exciting-monster-1585647870-cluster"
[ℹ]  deploying stack "eksctl-exciting-monster-1585647870-cluster"
[ℹ]  building nodegroup stack "eksctl-exciting-monster-1585647870-nodegroup-ng-22e7fed7"
[ℹ]  --nodes-min=2 was set automatically for nodegroup ng-22e7fed7
[ℹ]  --nodes-max=2 was set automatically for nodegroup ng-22e7fed7
[ℹ]  deploying stack "eksctl-exciting-monster-1585647870-nodegroup-ng-22e7fed7"
[✔]  all EKS cluster resources for "exciting-monster-1585647870" have been created
[✔]  saved kubeconfig as "/home/tammach/.kube/config"
[ℹ]  adding identity "arn:aws:iam::363193625107:role/eksctl-exciting-monster-158564787-NodeInstanceRole-10RLPVXJF1K8I" to auth ConfigMap
[ℹ]  nodegroup "ng-22e7fed7" has 0 node(s)
[ℹ]  waiting for at least 2 node(s) to become ready in "ng-22e7fed7"
[ℹ]  nodegroup "ng-22e7fed7" has 2 node(s)
[ℹ]  node "ip-192-168-56-48.us-east-2.compute.internal" is ready
[ℹ]  node "ip-192-168-68-250.us-east-2.compute.internal" is ready
[ℹ]  kubectl command should work with "/home/tammach/.kube/config", try 'kubectl get nodes'
[✔]  EKS cluster "exciting-monster-1585647870" in "us-east-2" region is ready

$ kgno                          
NAME                                           STATUS   ROLES    AGE   VERSION
ip-192-168-56-48.us-east-2.compute.internal    Ready    <none>   13m   v1.14.9-eks-1f0ca9
ip-192-168-68-250.us-east-2.compute.internal   Ready    <none>   13m   v1.14.9-eks-1f0ca9
```

</details>

<details>
<summary>Create new cluster with 1.13 </summary>

```
./eksctl create cluster --version=1.13
[ℹ]  eksctl version 0.17.0-dev+1c03c20c.2020-03-22T21:53:19Z
[ℹ]  using region us-east-2
[ℹ]  setting availability zones to [us-east-2a us-east-2b us-east-2c]
[ℹ]  subnets for us-east-2a - public:192.168.0.0/19 private:192.168.96.0/19
[ℹ]  subnets for us-east-2b - public:192.168.32.0/19 private:192.168.128.0/19
[ℹ]  subnets for us-east-2c - public:192.168.64.0/19 private:192.168.160.0/19
[ℹ]  nodegroup "ng-25c5b791" will use "ami-0c75489b2d6bb5afc" [AmazonLinux2/1.13]
[ℹ]  using Kubernetes version 1.13
[ℹ]  creating EKS cluster "scrumptious-monster-1584875597" in "us-east-2" region with un-managed nodes
[ℹ]  will create 2 separate CloudFormation stacks for cluster itself and the initial nodegroup
[ℹ]  if you encounter any issues, check CloudFormation console or try 'eksctl utils describe-stacks --region=us-east-2 --cluster=scrumptious-monster-1584875597'
[ℹ]  CloudWatch logging will not be enabled for cluster "scrumptious-monster-1584875597" in "us-east-2"
[ℹ]  you can enable it with 'eksctl utils update-cluster-logging --region=us-east-2 --cluster=scrumptious-monster-1584875597'
[ℹ]  Kubernetes API endpoint access will use default of {publicAccess=true, privateAccess=false} for cluster "scrumptious-monster-1584875597" in "us-east-2"
[ℹ]  2 sequential tasks: { create cluster control plane "scrumptious-monster-1584875597", create nodegroup "ng-25c5b791" }
[ℹ]  building cluster stack "eksctl-scrumptious-monster-1584875597-cluster"
[ℹ]  deploying stack "eksctl-scrumptious-monster-1584875597-cluster"
[ℹ]  building nodegroup stack "eksctl-scrumptious-monster-1584875597-nodegroup-ng-25c5b791"
[ℹ]  --nodes-min=2 was set automatically for nodegroup ng-25c5b791
[ℹ]  --nodes-max=2 was set automatically for nodegroup ng-25c5b791
[ℹ]  deploying stack "eksctl-scrumptious-monster-1584875597-nodegroup-ng-25c5b791"
[✔]  all EKS cluster resources for "scrumptious-monster-1584875597" have been created
[✔]  saved kubeconfig as "/home/tammach/.kube/config"
[ℹ]  adding identity "arn:aws:iam::363193625107:role/eksctl-scrumptious-monster-158487-NodeInstanceRole-16IJ011D9HISU" to auth ConfigMap
[ℹ]  nodegroup "ng-25c5b791" has 0 node(s)
[ℹ]  waiting for at least 2 node(s) to become ready in "ng-25c5b791"
[ℹ]  nodegroup "ng-25c5b791" has 2 node(s)
[ℹ]  node "ip-192-168-49-243.us-east-2.compute.internal" is ready
[ℹ]  node "ip-192-168-88-28.us-east-2.compute.internal" is ready
[ℹ]  kubectl command should work with "/home/tammach/.kube/config", try 'kubectl get nodes'
[✔]  EKS cluster "scrumptious-monster-1584875597" in "us-east-2" region is ready

$ kgno
NAME                                           STATUS   ROLES    AGE   VERSION
ip-192-168-49-243.us-east-2.compute.internal   Ready    <none>   77s   v1.13.12-eks-879c36
ip-192-168-88-28.us-east-2.compute.internal    Ready    <none>   80s   v1.13.12-eks-879c36
```

</details>

<details>
<summary>Create new cluster with 1.15 </summary>

```
$ ./eksctl create cluster --version 1.15                     
[ℹ]  eksctl version 0.17.0-dev+fe5ebf04.2020-03-31T20:43:57Z
[ℹ]  using region us-east-2
[ℹ]  setting availability zones to [us-east-2a us-east-2c us-east-2b]
[ℹ]  subnets for us-east-2a - public:192.168.0.0/19 private:192.168.96.0/19
[ℹ]  subnets for us-east-2c - public:192.168.32.0/19 private:192.168.128.0/19
[ℹ]  subnets for us-east-2b - public:192.168.64.0/19 private:192.168.160.0/19
[ℹ]  nodegroup "ng-1531807c" will use "ami-0da2b71057dfd9557" [AmazonLinux2/1.15]
[ℹ]  using Kubernetes version 1.15
[ℹ]  creating EKS cluster "beautiful-wardrobe-1585649591" in "us-east-2" region with un-managed nodes
[ℹ]  will create 2 separate CloudFormation stacks for cluster itself and the initial nodegroup
[ℹ]  if you encounter any issues, check CloudFormation console or try 'eksctl utils describe-stacks --region=us-east-2 --cluster=beautiful-wardrobe-1585649591'
[ℹ]  CloudWatch logging will not be enabled for cluster "beautiful-wardrobe-1585649591" in "us-east-2"
[ℹ]  you can enable it with 'eksctl utils update-cluster-logging --region=us-east-2 --cluster=beautiful-wardrobe-1585649591'
[ℹ]  Kubernetes API endpoint access will use default of {publicAccess=true, privateAccess=false} for cluster "beautiful-wardrobe-1585649591" in "us-east-2"
[ℹ]  2 sequential tasks: { create cluster control plane "beautiful-wardrobe-1585649591", create nodegroup "ng-1531807c" }
[ℹ]  building cluster stack "eksctl-beautiful-wardrobe-1585649591-cluster"
[ℹ]  deploying stack "eksctl-beautiful-wardrobe-1585649591-cluster"
[ℹ]  building nodegroup stack "eksctl-beautiful-wardrobe-1585649591-nodegroup-ng-1531807c"
[ℹ]  --nodes-min=2 was set automatically for nodegroup ng-1531807c
[ℹ]  --nodes-max=2 was set automatically for nodegroup ng-1531807c
[ℹ]  deploying stack "eksctl-beautiful-wardrobe-1585649591-nodegroup-ng-1531807c"
[✔]  all EKS cluster resources for "beautiful-wardrobe-1585649591" have been created
[✔]  saved kubeconfig as "/home/tammach/.kube/config"
[ℹ]  adding identity "arn:aws:iam::363193625107:role/eksctl-beautiful-wardrobe-1585649-NodeInstanceRole-QWPJJADIXEZB" to auth ConfigMap
[ℹ]  nodegroup "ng-1531807c" has 0 node(s)
[ℹ]  waiting for at least 2 node(s) to become ready in "ng-1531807c"
[ℹ]  nodegroup "ng-1531807c" has 2 node(s)
[ℹ]  node "ip-192-168-60-88.us-east-2.compute.internal" is ready
[ℹ]  node "ip-192-168-81-224.us-east-2.compute.internal" is ready
[ℹ]  kubectl command should work with "/home/tammach/.kube/config", try 'kubectl get nodes'
[✔]  EKS cluster "beautiful-wardrobe-1585649591" in "us-east-2" region is ready

$ kgno
NAME                                           STATUS   ROLES    AGE   VERSION
ip-192-168-60-88.us-east-2.compute.internal    Ready    <none>   88s   v1.15.10-eks-bac369
ip-192-168-81-224.us-east-2.compute.internal   Ready    <none>   87s   v1.15.10-eks-bac369

```

</details>

### Checklist
- [ ] Added tests that cover your change (if possible)
- [ ] Added/modified documentation as required (such as the `README.md`, or the `site/content` directory)
- [x] Manually tested
- [ ] Added labels for change area (e.g. `area/nodegroup`), target version (e.g. `version/0.12.0`) and kind (e.g. `kind/improvement`)
- [x] Make sure the title of the PR is a good description that can go into the release notes

### CODE REVIEW

**Comment by sayboras** [comment]
As there is a change in docker build image hash, will need help from maintainer to build and push docker image.

**Comment by sayboras** [comment]
@martina-if @cPu1 @marccarre need your input if it's aligning with what you are planning.

File: .circleci/config.yml [hunk 1]
```diff
@@ -40,12 +40,12 @@ commands:
...
               --volume=$HOME/go/pkg/mod/:/go/pkg/mod \
               --volume=$HOME/project/.ssh:/root/.ssh \
-              weaveworks/eksctl-build:$(cat .docker/image_tag) << parameters.cmd >>
+              sayboras/eksctl-build:$(cat .docker/image_tag) << parameters.cmd >>
           no_output_timeout: << parameters.timeout >>
   docker-pull-build-image:
     description: Docker pull image
     steps:
-      - run: docker pull "weaveworks/eksctl-build:$(cat .docker/image_tag)"
+      - run: docker pull "sayboras/eksctl-build:$(cat .docker/image_tag)"
```
Comment (marccarre) [inline .circleci/config.yml]: Is using `sayboras` as the Docker org. expected? (The commit introducing the change says "_Update image name for testing purpose_" so I am guessing this is an omission & needs to be removed before we merge?)

Comment (sayboras) [reply .circleci/config.yml]: @marccarre you are right. I already updated docker hash, but I don't have the permission to push to `weaveworks` docker hub, so just use my docker account for testing purpose in CI.

Might need one of maintainers to actually check out the branch and update Docker image. Thanks.

Comment (sayboras) [reply .circleci/config.yml]: Let me revert this change, however the build will fail later :)

**Comment by sayboras** [comment]
@marccarre @martina-if thanks for your help, I will verify the integration test tomorrow :)

**Comment by sayboras** [comment]
Integration test is running fine today https://circleci.com/gh/weaveworks/eksctl/7147?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link :)

**Pull request merged**